Add common properties argument to WSGIApplication wrapper

aaronkensci opened the pull request:
Our Flask app cannot log common properties/dimensions when it uses the default WSGIApplication wrapper. I have updated the wrapper construct to take in a new common_properties object. This is similar to the code used in the NodeJS Application Insight Telemetry Logger: https://github.com/Microsoft/ApplicationInsights-node.js/blob/develop/Library/TelemetryClient.ts#L36

These common properties are then logged for every track_request made by the WSGIApplication wrapper.

SergeyKanzhelev commented:
Can you please add an entry into changelog https://github.com/Microsoft/ApplicationInsights-Python/blob/master/CHANGELOG.md so release notes will be easier to write down.

SergeyKanzhelev commented on applicationinsights/requests/WSGIApplication.py:
```diff
@@ -49,6 +49,7 @@ def __init__(self, instrumentation_key, wsgi_application, *args, **kwargs):
         self.client = applicationinsights.TelemetryClient(instrumentation_key, telemetry_channel)
         self.client.context.device.type = "PC"
         self._wsgi_application = wsgi_application
+        self._common_properties = kwargs.pop('common_properties', {})
```
yes, it will be great if you can add a note into README.md. I see that `DESCRIPTION.rst` is out of sync with readme. @OsvaldoRosado @jjjordanmsft do you know the history here? From looks of it I'd say readme should just point to DESCRIPTION.rst instead of duplicating the context

jjjordanmsft commented:
I'd say just update README.md for now and while we ought to address the duplication, let's do so in another issue (#89).

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/ApplicationInsights-Python?pullRequest=88) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/ApplicationInsights-Python?pullRequest=88) before we can accept your contribution.<br/><br/>:x: aaronkensci [sign now](https://cla.opensource.microsoft.com/Microsoft/ApplicationInsights-Python?pullRequest=88)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/ApplicationInsights-Python?pullRequest=88) it.</sub>

aaronkensci commented:
@SergeyKanzhelev @jjjordanmsft 
I have updated the README and WSGIApplication documentation to surface the new functionality. I added the change to the '0.11.5' changelog version. Please edit it if it should be in the .4 version.

SergeyKanzhelev commented on CHANGELOG.md:
```diff
@@ -1,9 +1,10 @@
 # Changelog
 
+## 0.11.5
+- Add common properties argument to WSGIApplication initialization.
```
We haven't shipped `11.4` yet. So it should be reported under previous title. I'll merge and fix this
